services/workflows-service/src/workflow/hook-callback-handler.service.ts

```diff
@@ -153,27 +162,29 @@ export class HookCallbackHandlerService {
       firstIssue: kycDocument['firstIssue'],
     };
     const issuer = {
-      additionalDetails: additionalIssuerInfor,
+      additionalInfo: additionalIssuerInfor,
       country: kycDocument['country'],
       name: kycDocument['issuedBy'],
-      issuingVersion: kycDocument['issueNumber'],
       city: kycDocument['placeOfIssue'],
     };
     return issuer;
   }
 
-  formatPages(data: AnyRecord) {
+  async formatPages(data: AnyRecord) {
     const documentImages: AnyRecord[] = [];
     for (const image of data.images as { context?: string; content: string }[]) {
       const tmpFile = tmp.fileSync().name;
       const base64ImageContent = image.content.split(',')[1];
-      const data = Buffer.from(base64ImageContent as string, 'base64');
-      fs.writeFileSync(tmpFile, data);
+      const buffer = Buffer.from(base64ImageContent as string, 'base64');
+      const fileType = await fromBuffer(buffer);
```

Comment:
Its a bit conusing when using it like that, consider changing to `import { fromBuffer as getFileTypeFromBuffer} from 'file-type';`